docsets/ns-3/docset.json

```diff
@@ -12,6 +12,10 @@
 		{ 
 			"version": "3.25",
 			"archive": "versions/3.25/ns-3.tgz"
+		},	
+		{ 
+			"version": "3.26",
+			"archive": "versions/3.26/ns-3.tgz"
```

Comment:
Newer versions should be at the top of this list.

Also, you need to add the actual `tgz` for this version. Moving the `txt` file doesn't work.